indexer-app/src/cmr/indexer/data/concepts/keyword_util.clj

```diff
@@ -34,13 +42,71 @@
 
 (defn- contact-person->keywords
   "Converts a compound field into a vector of terms for keyword searches."
-  [data]
+  [contact-person]
   (let [{first-name :FirstName
          last-name :LastName
-         roles :Roles} data]
+         roles :Roles} contact-person]
     (concat [first-name last-name]
             roles)))
 
+(defn- get-contact-persons
+  "Retrieve a vector of contact persons."
+  [data]
```

Comment:
I think it's better to keep the original "collection" so that we know what "data" this represents.